syntax = "proto2";
message ServerInfo_User {
    enum UserLevelFlag {
        IsNothing = 0;
        IsUser = 1;
        IsRegistered = 2;
        IsModerator = 4;
        IsAdmin = 8;
        IsJudge = 16;
    };
    optional string name = 1;
    optional uint32 user_level = 2;
    optional string address = 3;
    optional string real_name = 4;
    // gender = 5; // obsolete
    optional string country = 6;
    optional bytes avatar_bmp = 7;
    optional sint32 id = 8 [default = -1];
    optional sint32 server_id = 9 [default = -1];
    optional uint64 session_id = 10;
    optional uint64 accountage_secs = 11;
    optional string email = 12;
    optional string clientid = 13;
    optional string privlevel = 14;
}
